pkg/controllers/manila/openstack.go

```diff
@@ -14,18 +14,14 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/sharedfilesystems/v2/sharetypes"
 	"github.com/gophercloud/utils/openstack/clientconfig"
 	"github.com/openshift/csi-driver-manila-operator/pkg/util"
-	"github.com/openshift/library-go/pkg/operator/v1helpers"
 	"sigs.k8s.io/yaml"
 )
 
 type openStackClient struct {
 	cloud *clientconfig.Cloud
 }
 
-func NewOpenStackClient(
-	cloudConfigFilename string,
-	informers v1helpers.KubeInformersForNamespaces,
```

Comment:
I don't know why this was passed as noting was using it. Perhaps this was seen as a potential issue during implementation but was never addressed?